tests/util/test_object.py

```diff
@@ -233,3 +233,40 @@ def test_merge_v2():
         "baz": "Baz",
         "test": {"attr1": "Foo Bar", "attr2": None, "attr3": ["1", "2"]},
     }
+
+
+def test_without_attr_returns_list_without_the_attr():
+    """
+    Test that without_attr() returns a list without the attr
+    """
+    dict1 = {
+        "Name": "Geeks",
+        "Gender": "Male"
+    }
+    dict2 = {
+        "Gender": "Male"
+    }
+    assert without_attr(dict1, ["Name"]) == dict2
+
+def test_without_attr_return_list_without_the_attr_for_nested_dict():
+    """
+    Test that without_attr() returns a list without the attr
+    for nested dictionaries
+    """
+    dict1 = {
+        "Name": "Geeks",
+        "Gender": "Male",
+        "Age": "55",
+        "Address": {
+            "Street": "Charkhal",
+            "District": "Kathmandu",
+        }
+    }
+    dict1 = {
```

Comment:
this should be dict2 right?

Reply:
Yes, thanks.

Reply:
@sanjevShakya It passed due to variable override. :D